go/libkb/loaduser.go

```diff
@@ -360,3 +362,64 @@ func LoadUserPlusKeys(g *GlobalContext, uid keybase1.UID, cacheOK bool) (keybase
 
 	return up, nil
 }
+
+func checkForNewTracksIfSelf(arg LoadUserArg, idtable *IdentityTable) error {
+	arg.G().Log.Debug("+ checkForNewTracksIfSelf")
+	defer arg.G().Log.Debug("- checkForNewTracksIfSelf")
+
+	// If we're loading a user other than ourselves, short-circuit.
+	if !arg.Self || idtable == nil {
+		return nil
+	}
+
+	// Get the our last seen seqno from the local DB.
+	dbkey := DbKeyUID(DBLastSeenSeqno, arg.UID)
```

Comment:
Shouldn't this be available as a by-product as the loading process?  Do we need to keep track of this separately?